packages/gdl-frontend/components/GlobalMenu/index.js

```diff
@@ -84,7 +94,11 @@ class GlobalMenu extends React.Component<Props, State> {
                   </ListItem>
                 )}
               </SelectBookLanguage>
```

Comment:
You need to add the same functionality to `<SelectBookLanguage ... >` for it to work. E.g:
```javascript
<SelectBookLanguage
onSelectLanguage={onClose}
disableParentSwipe={this.disableSwipe}
enableParentSwipe={this.enableSwipe}
>
```

And then trigger those functions in the same methods in `SelectBookLanguage.js` E.g:
```javascript
handleCloseMenu = () => {
  this.setState({ showMenu: false });
  this.props.enableParentSwipe();
};
```

Reply:
Nice catch! Fixed 👍